.ci_support/linux_aarch64_python3.9.____cpython.yaml

```diff
@@ -0,0 +1,29 @@
+BUILD:
+- aarch64-conda_cos7-linux-gnu
```

Comment:
Why is this being added manually? Did you try [re-rendering]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#dev-update-rerender )?

Reply:
Thanks, I will take a look at what that means and try to re-render.
Sorry, I'm a beginner in conda-forge maintenance.

Reply:
xref: https://github.com/conda-forge/bottleneck-feedstock/issues/37